### ISSUE TITLE
Unable to use custom config file.

### ISSUE THREAD

**Issue by aharris** [danielhusar/gulp-stylint#4]
I am trying to use a custom .stylintrc however, I cannot get it to see the changes.

My task:

```
gulp.task('stylint', function() {
  var stylint = require('gulp-stylint');
  return gulp.src(['**/*.styl', '!./styl/lib/**/*.styl', '!./node_modules/**/*.styl'])
    .pipe(stylint('.stylintrc'))
});
```

.stylintrc is located in the same directory as my gulpfile.

Am I passing this parameter incorrectly?

**Comment by danielhusar** [comment]
Hey,

yep, you have to pass it like this:

```
gulp.task('stylint', function() {
  var stylint = require('gulp-stylint');
  return gulp.src(['**/*.styl', '!./styl/lib/**/*.styl', '!./node_modules/**/*.styl'])
    .pipe(stylint({config: '.stylintrc'}))
});
```

**Comment by aharris** [comment]
Ahh... thank you so much.

**Comment by diessica** [comment]
I stumbled upon the same issue and didn't even notice why my config file wasn't working. It'd be great if it worked just like we are used to, as gulp-jshint does: `.pipe(__.jshint('.jshintrc'))`

**Comment by danielhusar** [comment]
Hey,

This plugin support only two options: `config` and `strict mode`as all other styling options are redundant here.
I can make a shortcut if string is passed as option to use it as config path but Im not sure this brings much benefits, as its quite good documented, and you have even readme with example how to pass custom config.

**Comment by diessica** [comment]
yeah, I know it and even agree with you that it is well-documented. I was just saying it is like intuitive because of JSHint, you know :P

**Comment by diessica** [comment]
yeah, I know it and even agree with you that it is well-documented. I was just saying it is like intuitive because of JSHint, you know :P the shortcut would be great anyway!

**Comment by SimenB** [comment]
I implemented `rcloader` (which accepts a string as first option as path), and also picks up rcfiles automatically, gets the correct one if nested etc. I ended up ripping it out though as stylint itself has some weird handling of options I didn't dig deeper into, and stylint also looks up `.stylintrc` automatically.
Might give it another look, but I think some changes to how stylint handles options should be done first.
